README.md

```diff
@@ -19,9 +19,95 @@ Assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running the demo on Windows
+
+Prerequisites:
+
+- [Docker desktop](https://docs.docker.com/docker-for-windows/install/)
+
+### Step 1
+
+First test that the docker installation is working. Open a Powershell prompt (press Windows+S and type Powershell) and run:
+
+```shell
+docker run hello-world
+```
+
+This should show a message that your Docker installation is working correctly. If so, we can proceed to the installation of evidence tool, otherwise we suggest to check the [Docker troubleshooting page](https://docs.docker.com/docker-for-windows/troubleshoot/).
+
+### Step 2
+
+[Download](https://github.com/ADAH-EviDENce/evidence/archive/master.zip) a copy of evidence archive and extract its contents on your machine.
+
+### Step 3
+
+Open a Powershell prompt:
+Change your current working directory to where you extracted the files. For instance:
+
+```shell
+cd C:\Users\JohnDoe\Downloads\evidence-master\evidence-master
```

Comment:
```suggestion
cd C:\Users\JohnDoe\Downloads\evidence
```
is probably easier. Note this will also affect the container names below.

Reply:
This is the path you get when you use the Windows extractor with default settings. So I would like to keep it.